pkg/antctl/handlers/agentinfo.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/clusterinformation/v1beta1"
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+var _ Factory = new(AgentInfo)
+
+// AgentInfo is the implementation of the Factory interface for the agent-info command.
+type AgentInfo struct{}
```

Comment:
Humm...
Question - what is the convention of this handler struct. Do we assume it includes the real result data and how about other commands? If so, should we add AntreaAgentInfo into the struct?

Reply:
I have put an example in the PR description.

This struct doesn't contain data or result. It is only for command handler below:
`func (v *AgentInfo) Handler(aq monitor.AgentQuerier, cq monitor.ControllerQuerier) http.HandlerFunc`